### ISSUE TITLE
'Error parsing LASResponse Text as JSON'

### ISSUE THREAD

**Issue by karlmsmith** [NOAA-PMEL/LAS#107]
**Reported by jonathan.s.callahan on 31 Jul 2007 16:40 UTC**
This error occurs on IE only when asking for a SlideSorter XY product for

Datasets > by Dataset Name > COADS monthly time series (1854-1993)

even though IE works fine with COADS_climatology.

Migrated-From: http://dunkel.pmel.noaa.gov/trac/las/ticket/103

**Comment by karlmsmith** [comment]
**Comment by jonathan.s.callahan on 31 Jul 2007 16:42 UTC**
This happens on Firefox as well and it appears that the LASGetGridResponse is getting an HTML response instead of a JSON response.

**Comment by karlmsmith** [comment]
**Comment by jonathan.s.callahan on 31 Jul 2007 17:09 UTC**
This ticket depends on ticket 104.

**Comment by karlmsmith** [comment]
**Modified by jonathan.s.callahan on 1 Aug 2007 17:38 UTC**

**Comment by karlmsmith** [comment]
**Comment by jonathan.s.callahan on 1 Aug 2007 17:48 UTC**
This ticket is being resolved as a duplicate of #104.
